frontend/ui/data/css_tweaks.lua

```diff
@@ -338,38 +347,89 @@ p {
+                priority = 2, -- so it can override 'paragraph_indent'
+                css = [[p:first-child { text-indent: 0 !important; }]],
+            },
+            {
+                id = "paragraph_following_no_indent";
+                title = _("No indentation on following paragraphs"),
+                description = _("Do not indent the first line of following paragraphs, but leave the first paragraph of its container untouched."),
+                priority = 2, -- so it can override 'paragraph_indent'
+                css = [[p + p { text-indent: 0 !important; }]],
+            },
         },
         {
-            id = "paragraph_whitespace";
             title = _("Spacing between paragraphs"),
-            description = _("Add a line of whitespace between paragraphs."),
-            css = [[p + p { margin-top: 1em !important; }]],
-        },
-        {
-            id = "paragraph_whitespace_half";
-            title = _("Spacing between paragraphs (half)"),
-            description = _("Add half a line of whitespace between paragraphs."),
-            css = [[p + p { margin-top: .5em !important; }]],
+            {
+                id = "paragraph_whitespace";
+                title = _("Spacing between paragraphs"),
+                description = _("Add a line of whitespace between paragraphs."),
+                priority = 5, -- Override "Paragraphs margins and paddings" below
+                css = [[p + p { margin-top: 1em !important; }]],
+            },
+            {
+                id = "paragraph_whitespace_half";
+                title = _("Spacing between paragraphs (half)"),
+                description = _("Add half a line of whitespace between paragraphs."),
+                priority = 5,
+                css = [[p + p { margin-top: .5em !important; }]],
+            },
+            {
+                id = "paragraph_no_whitespace";
+                title = _("No spacing between paragraphs"),
+                description = _("No whitespace between paragraphs is the default, but it may be overridden by publisher styles. This will re-enable it for paragraphs and list items."),
+                priority = 5,
+                css = [[p, li { margin-top: 0 !important; margin-bottom: 0 !important; }]],
+            },
         },
         {
-            id = "paragraph_no_whitespace";
-            title = _("No spacing between paragraphs"),
-            description = _("No whitespace between paragraphs is the default, but it may be overridden by publisher styles. This will re-enable it for paragraphs and list items."),
-            css = [[p, li { margin-top: 0 !important; margin-bottom: 0 !important; }]],
+            title = _("Paragraphs margins and paddings"),
+            {
+                id = "margin_horizontal_most_0";
+                title = _("Ignore paragraphs horizontal margins"),
```

Comment:
This collection sounds a bit awkward, mainly because of the adjective order.

```suggestion
                title = _("Ignore horizontal paragraph margins"),
```